index.bs

```diff
@@ -0,0 +1,11 @@
+<pre class='metadata'>
+Title: Web Extensions Draft
+Shortname: webextensions
+Level: 1
+Status: w3c/ED
+Group: w3c/webextensions

```

Comment:
s|w3c/webextensions|WECG|

Reply:
Note that this will require a corresponding change to Bikeshed, which I'll submit a PR for today.

Reply:
https://github.com/tabatkins/bikeshed/pull/2111

Reply:
nota bene https://github.com/tabatkins/bikeshed/pull/2111#issuecomment-880984568